activities.json

```diff
@@ -933,6 +933,19 @@
     "title": "Mixed Content",
     "url": "https://w3c.github.io/webappsec-mixed-content/"
   },
+  {
+    "ciuName": null,
+    "description": "The Network Error Logging spec enables website to declare a reporting policy that can be used to report encountered network errors that prevented it from successfully loading its requested resources.",
+    "id": "network-error-logging",
+    "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1145235",
+    "mozPosition": "negative",
+    "mozPositionDetail": "The API allows access to new kinds of network errors in a way that may leave private user info exposed. In particular, we question the value of collecting this information on a per-user basis when there is no framework for consent or transparency. Furthermore, the specification does not seem to track changes to the Reporting API it builds upon and seems effectively unmaintained.",
```

Comment:
I think that the key piece of feedback here is that the reports contain information that we don't currently provide to sites, which might include information that people consider private.

There isn't much point in talking about consent here: there is no sense in us asking for permission to expose this sort of information.  Transparency is maybe better, but it depends on there being a system for accountability.  I'd mention neither.

Maybe "This API enables the collection of user-specific information that sites might not otherwise be able to observe, which includes information that might be private in some circumstances.  Furthermore, [your point about the spec]"

Reply:
yup, I agree this might not make a lot of sense to talk about consent. I adopted your phrase in the next commit.